src/components/Organization/Dashboard/ProcessCard.tsx

```diff
@@ -11,13 +16,76 @@ const ProcessCard = () => {
   if (!election) return null
```

Comment:
Do this and you won't need to use `as any`. Anyways, you should avoid using `as any` at all costs...  you could have used `as PublishedElection`, which would have been a bit better, but would still have the same issue we're having here (an invalid election will crash the app).

```suggestion
  if (!election || election instanceof InvalidElection) return null
```